src/ring_theory/jacobson.lean

```diff
@@ -121,6 +121,26 @@ lemma is_jacobson_iso (e : R ≃+* S) : is_jacobson R ↔ is_jacobson S :=
 ⟨λ h, @is_jacobson_of_surjective _ _ _ _ h ⟨(e : R →+* S), e.surjective⟩,
   λ h, @is_jacobson_of_surjective _ _ _ _ h ⟨(e.symm : S →+* R), e.symm.surjective⟩⟩
 
+lemma is_jacobson_of_is_integral [algebra R S] (hRS : algebra.is_integral R S)
+  (hR : is_jacobson R) : is_jacobson S :=
+begin
+  rw is_jacobson_iff_prime_eq,
+  introsI P hP,
+  by_cases hP_top : comap (algebra_map R S) P = ⊤,
+  { simp [comap_eq_top_iff.1 hP_top] },
+  { haveI : nontrivial (comap (algebra_map R S) P).quotient := quotient.nontrivial hP_top,
+    rw jacobson_eq_iff_jacobson_quotient_eq_bot,
+    refine eq_bot_of_comap_eq_bot (is_integral_quotient_of_is_integral hRS) _,
+    erw [eq_bot_iff, ← jacobson_eq_iff_jacobson_quotient_eq_bot.1 ((is_jacobson_iff_prime_eq.1 hR)
+      (comap (algebra_map R S) P) (comap_is_prime _ _)), comap_Inf'],
```

Comment:
So we can turn `erw` into `rw`: could you make a lemma `comap_jacobson` that is `comap_Inf'` specialized to `ideal.jacobson`?